src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -2251,6 +2251,17 @@ public static FormulaValue Error(IRContext irContext, FormulaValue[] args)
                     case OptionSetValue osv:
                         errorKind = (ErrorKind)Convert.ToInt32(osv.ExecutionValue, CultureInfo.InvariantCulture);
                         break;
+                    case StringValue sv:
+                        if (!int.TryParse(sv.Value, out var kindInt))
+                        {
+                            return CommonErrors.RuntimeTypeMismatch(irContext);
+                        }
+
+                        errorKind = (ErrorKind)kindInt;
+                        break;
+                    case BooleanValue bv:
+                        errorKind = (ErrorKind)Convert.ToInt32(bv.Value, CultureInfo.InvariantCulture);
+                        break;

```

Comment:
I know this is the way it's currently implemented, but can this be solved in the binder, by using coercion nodes? For example, something like `Error( [{Kind:ErrorKind.Div0}, {Kind:"14"}] )` be IR'd as

    Error:N(
        Table:*[Kind:n](
            {Kind: OptionSetToNumber:n(ScopeAccess(Scope1,Kind))},
            {Kind: Value:n("14")}))